src/impl/dtlstransport.hpp

```diff
@@ -70,7 +70,35 @@ class DtlsTransport : public Transport, public std::enable_shared_from_this<Dtls
 	static ssize_t WriteCallback(gnutls_transport_ptr_t ptr, const void *data, size_t len);
 	static ssize_t ReadCallback(gnutls_transport_ptr_t ptr, void *data, size_t maxlen);
 	static int TimeoutCallback(gnutls_transport_ptr_t ptr, unsigned int ms);
-#else
+
+#elif USE_MBEDTLS
+	std::mutex mMutex;
+
+	mbedtls_entropy_context mEntropy;
+	mbedtls_ctr_drbg_context mDrbg;
+	mbedtls_ssl_config mConf;
+	mbedtls_ssl_context mSsl;
+
+	uint32_t mFin_ms, mInt_ms;
```

Comment:
I'd prefer `mFinMs` and `mIntMs` to prevent mixing camel-case and snake-case.

Reply:
Fixed! Renamed